main.go

```diff
@@ -27,4 +38,22 @@ func main() {
 		h.RegisterPull()
 	}
 	h.Run()
+	// h.RunOnAddr(viper.GetString("host"))
```

Comment:
FIXME LATER: not as simple as it looks. 127.0.0.1 is not the correct IP to listen. Instead, we should listen for all IPs in the internal overlay network.

Reply:
That's the idea, yes. No need for authentication, because if they have control of ada, it's just a question of time 'til they find the authtoken to restart a container.